### PR TITLE
Basic KV Store Service

### PR DESCRIPTION
**Pull request by refs**

### CODE REVIEW

File: pkg/service/v0/option.go [hunk 1]
```diff
@@ -35,17 +34,26 @@ func Logger(val log.Logger) Option {
...
-// Middleware provides a function to set the middleware option.
-func Middleware(val ...func(http.Handler) http.Handler) Option {
+func Nodes(val *config.Config) Option {
 	return func(o *Options) {
-		o.Middleware = val
+		o.Config = val
 	}
 }
 
+func Config(val *config.Config) Option {
```
Comment (micbar) [inline pkg/service/v0/option.go:55]: Codacy found an issue: [exported function Config should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: pkg/service/v0/option.go [hunk 2]
```diff
@@ -35,17 +34,26 @@ func Logger(val log.Logger) Option {
...
 
-// Config provides a function to set the config option.
-func Config(val *config.Config) Option {
+func Database(val *config.Config) Option {
+	return func(o *Options) {
+		o.Config = val
+	}
+}
+
+func Table(val *config.Config) Option {
```
Comment (micbar) [inline pkg/service/v0/option.go:43]: Codacy found an issue: [exported function Table should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: pkg/service/v0/option.go [hunk 3]
```diff
@@ -35,17 +34,26 @@ func Logger(val log.Logger) Option {
...
 	return func(o *Options) {
 		o.Config = val
 	}
 }
 
-// Middleware provides a function to set the middleware option.
-func Middleware(val ...func(http.Handler) http.Handler) Option {
+func Nodes(val *config.Config) Option {
```
Comment (micbar) [inline pkg/service/v0/option.go:49]: Codacy found an issue: [exported function Nodes should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: pkg/service/v0/option.go [hunk 4]
```diff
@@ -35,17 +34,26 @@ func Logger(val log.Logger) Option {
 	}
 }
 
-// Config provides a function to set the config option.
-func Config(val *config.Config) Option {
+func Database(val *config.Config) Option {
```
Comment (micbar) [inline pkg/service/v0/option.go:37]: Codacy found an issue: [exported function Database should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: .drone.star [hunk 5]
```diff
@@ -60,32 +60,11 @@ def testing(ctx):
...
-        'commands': [
-          'make staticcheck',
-        ],
-        'volumes': [
-          {
-            'name': 'gopath',
-            'path': '/srv/app',
-          },
-        ],
-      },
```
Comment (micbar) [inline .drone.star:76]: ok, seems interesting to change that in the boilerplate

File: pkg/service/v0/option.go (hunk 1 above)
Comment (micbar) [inline pkg/service/v0/option.go:55]: Codacy found an issue: [exported function Config should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: pkg/service/v0/option.go (hunk 2 above)
Comment (micbar) [inline pkg/service/v0/option.go:43]: Codacy found an issue: [exported function Table should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: pkg/service/v0/option.go (hunk 3 above)
Comment (micbar) [inline pkg/service/v0/option.go:49]: Codacy found an issue: [exported function Nodes should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: pkg/service/v0/option.go (hunk 4 above)
Comment (micbar) [inline pkg/service/v0/option.go:37]: Codacy found an issue: [exported function Database should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)

File: .drone.star (hunk 5 above)
Comment (refs) [reply .drone.star:76]: @micbar issue on ocis-boilr?

Comment (refs) [reply .drone.star:76]: perhaps I have to review my boilr local version, it might be outdated :)

**Review by butonic — APPROVED** [review]

**Pull request merged**